api/v1/editTaskTemplates/PKPEditTaskTemplateController.php

```diff
@@ -90,4 +93,50 @@ public function add(AddTaskTemplate $illuminateRequest): JsonResponse
         );
     }
 
+    /**
+     * GET /api/v1/editTaskTemplates
+     */
+    public function getMany(Request $request): JsonResponse
+    {
+        $context = $this->getRequest()->getContext();
+
+        // Start with our standard collector/scopes
+        $collector = Template::byContextId((int) $context->getId())
+            ->with('userGroups');
+
+        // Apply supported filters from query params via model scopes
+        foreach ($request->query() as $param => $val) {
+            switch ($param) {
```

Comment:
Can you take a look at the example of the params filtering in the submission controller: https://github.com/pkp/pkp-lib/blob/main/api/v1/submissions/PKPSubmissionController.php#L473?